VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts

```diff
@@ -51,7 +51,8 @@ import {
     router,
     userMediaService,
     userMediaStreamService,
-    videoWebService
+    videoWebService,
```

Comment:
Is this no longer used?

Reply:
I added videoCallService, as for videoWebService I guess there is somewhere used in the test

Reply:
Yeah I see. The wrong one being used on Line 121 threw me off